webapp/src/components/sidebar_buttons/index.js

```diff
@@ -4,27 +4,25 @@
 import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 
-import {list, updateRhsState, telemetry} from '../../actions';
+import {fetchAllIssue, updateRhsState, telemetry} from '../../actions';
 
 import SidebarButtons from './sidebar_buttons.jsx';
 
 function mapStateToProps(state) {
     return {
-        issues: state['plugins-com.mattermost.plugin-todo'].issues,
-        inIssues: state['plugins-com.mattermost.plugin-todo'].inIssues,
-        outIssues: state['plugins-com.mattermost.plugin-todo'].outIssues,
+        allIssues: state['plugins-com.mattermost.plugin-todo'].allIssues,
```

Comment:
I think it makes sense to keep the convention of feeding the component the issues the way we were doing before:

```js
const emptyArray = [];

function mapStateToProps(state) {
    const allIssues = getAllIssues(state); // from selectors.js
    return {
        myIssues: allIssues.my, // note that I changed the prop to be `myIssues`
        inIssues: allIssues.in,
        outIssues: allIssues.out,
```

Reply:
Though as mentioned elsewhere, we can use individual selectors for the in issues. The main reason I'm thinking this, is it simplifies what the component needs to worry about